presto-mongodb/pom.xml

```diff
@@ -189,7 +189,6 @@
                 <configuration>
                     <!-- integration tests take a very long time so only run them in the CI server -->
                     <parallel />
-                    <threadCount>1</threadCount>
```

Comment:
Hi @miniway! @losipiuk suggested that this line might actually be needed. Do you remember putting it on purpose? Were there any test failures when running in parallel? Please see the discussin in #5472.